6/README.md

```diff
@@ -0,0 +1,1078 @@
+		<td> DEV_STRING </td>
+		<td> DEVVAR_STRINGARRAY </td>
+		<td> OPERATOR </td>
+		<td> Get the list of devices using forwarded attribute(s) from specified device </td>
+	</tr>
+</table>
+
+## Database as a file
+For Device servers not able to access the Tango database (most of the time due to network route or security reason), 
+it is possible to start them using a file instead of a real database. This is done via the Device server command line option:  
+```-file=<file name>```  
+
+In this case, the Database SHALL handle the following functionality using the specified file:  
+* getting, setting and deleting class properties  
+* getting, setting and deleting device properties  
+* getting, setting and deleting class attribute properties  
+* getting, setting and deleting device attribute properties  
+
+The following set of limitations exist:  
+* no check that the same Device server is running twice
+* no Device or attribute alias name
+* in case of several Device servers running on the same host, the user must manually manage a list of already used network ports
+
+### File syntax
+The file is an ASCII file and SHALL follow a well-defined syntax with predefined keywords.
+
+Generally, comment starts with the ’#’ character and a blank line is ignored.
+
+#### Devices definition.
+DEVICE is the keyword to declare a device(s) definition sequence. The general syntax SHALL be   
+```<DS name>/<inst name>/DEVICE/<Class name>: dev1,dev2,dev3  ```
+Device(s) name can follow on next line if the last line character is ’\’
+
+#### Device property definition. 
+The general device property SHALL be  
+```<device name>-><property name>: <property value>```  
+In case of array, the array element delimiter is the character ’,’. Array definition can be splitted on several lines if the last line character is ’\’. Allowed characters after the ’:’ delimiter are space, tabulation or nothing.
+
+A device string property with special characters (spaces). The ’’ character is used to delimit the string
+
+#### Device attribute property definition. 
+The general device attribute property syntax SHALL be   
+```<device name>/<attribute name>-><property name>: <property value>```  
+Allowed characters after the ’:’ delimiter are space, tabulation or nothing.
+
+#### Class property definition. 
+The general class property syntax SHALL be  
+```CLASS/<class name>-><property name>: <property value>```  
+
+CLASS is the keyword to declare a class property definition. Allowed characters after the ’:’ delimiter are space, tabulation or nothing. 
+The ’’ characters around the property value are mandatory due to the ’/’ character contains in the property value.
+
+
+## Related documentation
+[Tango Controls Documentation 9.3.3](https://buildmedia.readthedocs.org/media/pdf/tango-controls/latest/tango-controls.pdf)
+
+[PyTango Documentation Release 8.1.6](http://www.esrf.eu/computing/cs/tango/pytango/v818/PyTango.pdf)
```

Comment:
This probably can be updated to the most recent one: https://readthedocs.org/projects/pytango/downloads/pdf/stable/